BackofficeBundle/Resources/public/coffee/orchestraLib.coffee

```diff
@@ -11,7 +11,7 @@ renderPageTitle = ->
 
 # ADD CUSTOM JARVIS WIDGET
 addCustomJarvisWidget = (widget) ->
-  $(widget).insertAfter($(".js-widget-title"))
+  $(widget).insertBefore($(".js-widget-title"))
```

Comment:
I think it's a trully big change that goes far further than the +/- problem. Do you have check that the other widgets are not broken ?
I'm very doubtfully about this big change.